src/pages/access_review/VoteSummary.js

```diff
@@ -155,7 +155,8 @@ export const VoteSummary = hh(
           padding: '1rem',
           textAlign: 'right',
           flexShrink: '0',
-          fontWeight: Theme.font.weight.regular}
+          fontWeight: Theme.font.weight.regular},
+        isRendered: this.props.finalDecision === "No decision"
```

Comment:
This seems like a very brittle condition. This text can easily change at some point and that will have a downstream effect here that will be difficult to track. I think it makes more sense to condition this on the state of the election. If the data access election is open, we should render the link. It doesn't make sense to send reminders in any other case (closed, canceled, approved, denied, etc.)